frontend/src/_ui/Drawer/DrawerFooter/index.jsx

```diff
@@ -80,12 +80,7 @@ function DrawerFooter({
     editForeignKeyInCreateTable,
   ]);
 
-  const isDrawerWithDocumentation =
-    initiator === 'CreateTableForm' ||
-    initiator === 'EditTableForm' ||
-    initiator === 'CreateColumnForm' ||
-    initiator === 'EditColumnForm' ||
-    initiator === 'ForeignKeyTableForm';
+  const isDrawerWithDocumentation = initiator === 'ForeignKeyTableForm';
```

Comment:
Don't we need to have links for other forms too to their corresponding doc as per design?